docs/Developers/Review.rst

```diff
@@ -105,6 +105,11 @@ For the Reviewers
 
 -  All the pull requests waiting to be reviewed can be found at :
    https://coala.io/review.
+-  Check the commit message.
```

Comment:
Maybe add "Check that the changes haven't been pushed from master branch" to/after this ^^ point - Else LGTM as well. 😄

Reply:
Then we have reviewers telling newcomers to close their PRs, even if the PR is good and can be merged quickly.

You've raised an important issue, which should be in this guide, but it needs to be a separate addition with lots of careful wording